INSS.EIIR.DataAccess/DatabaseScripts/getEiirIndex.sql

```diff
@@ -477,6 +483,7 @@ END AS tradingNames,
 	LEFT JOIN ci_ip cip ON insolvencyAppointment.ip_no = cip.ip_no
 	LEFT JOIN ci_ip_address cipa ON insolvencyAppointment.ip_no = cipa.ip_no
     LEFT JOIN ci_iva_case ivaCase ON ivaCase.case_no = inscase.case_no
+	LEFT JOIN #discharge discharge ON discharge.case_no = cases.CaseNo  and  discharge.indiv_no = cases.IndivNo  
```

Comment:
update keyword and to uppercase  to keep the readability and consistency of the proc